js/bitmax.js

```diff
@@ -348,41 +523,61 @@ module.exports = class bitmax extends Exchange {
         }
```

Comment:
The `request['n']` param for fetchOrderBook is not documented, and, apparently, it does not work. At least, it returns  more bids and asks than is specified in `n`:

- https://bitmax.io/api/pro/v1/depth?symbol=BTC-PERP&n=3